src/App/Actions/HerolistActions.ts

```diff
@@ -168,6 +168,11 @@ export const saveHero =
                  })
     }
 
+export const exportHeroAsRptok =
+  (id: string): ReduxAction =>
+  dispatch =>
+    dispatch (requestHeroExportAsRptok (id))
+
```

Comment:
Das hier könntest du einfacher schreiben, da du hier keinen State abfragst oder eine asynchrone Funktion ausführst:

```suggestion
export const exportHeroAsRptok = requestHeroExportAsRptok
```